### DISCUSSION TITLE
Winget or Microsoft Store?

### DISCUSSION THREAD

**Discussion by dxnnie** [ciderapp/Cider#605] [Q&A] [answered]
Is there a reason to use `winget` over the` Microsoft Store` version? Knowing that `Spicetify-CLI for Spotify` do not work with UWP version and `iTunes` has drastic performances issues, I started wonder if `Cider` had any relevant differences between versions.

**Comment by yazninja** [comment, marked as answer]
when using the winget version, you must expect bugs, because winget version are directly compiled from develop branch, whereas Microsoft Store version has been fletched out. Also buying Cider from the Microsoft Store shows your support to the hard-working devs.

**Reply by dxnnie** [reply]
> when using the winget version, you must expect bugs, because winget version are directly compiled from develop branch, whereas Microsoft Store version has been fletched out. Also buying Cider from the Microsoft Store shows your support to the hard-working devs.

Thanks.

I had brought the Microsoft Store version even before asking this. Devs are awesome. Now i'm +1 waiting on Apple musickit to being released.
